### ISSUE TITLE
No "Reset Controls" button.

### ISSUE THREAD

**Issue by IcingVR** [coop-deluxe/sm64coopdx#273]
**Describe the bug**
Not a bug, But there isn't a "Reset Controls" button

**To Reproduce**
1. Go to 'Controls Menu'
 
**Desktop (please complete the following information):**
 - OS: [Windows 10]
 - Browser [Firefox]
 - Version [10 22H2]

**Comment by frog8412** [comment]
yeah, currently there isnt a reset controls button, but you can do so by deleting sm64config.txt in %appdata%\sm64coopdx
